.github/workflows/plugins.yml

```diff
@@ -484,6 +484,17 @@ jobs:
         uses: ./.github/actions/testagent/logs
       - uses: codecov/codecov-action@v3
 
+  url:
+    strategy:
+      matrix:
+        node-version: ['18', '20', 'latest']
```

Comment:
Not sure it is working as expected. All plugin jobs for url - [url(18)](https://github.com/DataDog/dd-trace-js/actions/runs/11580977176/job/32240746741?pr=4836#step:3:304), [url(20)](https://github.com/DataDog/dd-trace-js/actions/runs/11580977176/job/32240747104?pr=4836#step:3:306) and [url(latest)](https://github.com/DataDog/dd-trace-js/actions/runs/11580977176/job/32240747522?pr=4836#step:3:302) - are running w/ node v18

Reply:
not, this `matrix` is not necessary, as at the end I am using `./.github/actions/plugins/test` action, I don't need this matrix, internally it is already running in each node version. I'll replace it.